LocationServices/LocationServices/Scenes/Settings/Subviews/MapStyle/Controller/MapStyleVC+CollectionView.swift

```diff
@@ -78,4 +97,19 @@ extension MapStyleVC: UICollectionViewDelegate, UICollectionViewDelegateFlowLayo
             self.viewModel.saveSelectedState(indexPath)
         }
     }
+    
+    func collectionView(_ collectionView: UICollectionView,
+                        layout collectionViewLayout: UICollectionViewLayout,
+                        minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
+        if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
I think we can replace it with one if - it would be more clear and elegance.
let device = UIDevice.current.
if device.userInterfaceIdiom == .ipad &&
   device.getDeviceOrientation() == .landscapeLeft {
return 50}
return 0